monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/utils/CommandDisplay.js

```diff
@@ -26,6 +26,22 @@ export default function commandDisplay(props) {
     }
   }, [props.commands]);
 
+  function handleKeyDown(event) {
+    let charCode = String.fromCharCode(event.which).toLowerCase();
+    if ((event.ctrlKey || event.metaKey) && charCode === 'c') {
+      props.onCopy();
+    }
+    if ((event.ctrlKey || event.metaKey) && charCode === 'a') {
+      event.preventDefault();
+      let codeElement = event.target.querySelector('code');
+      let selection = window.getSelection();
+      let range = document.createRange();
+      range.selectNode(codeElement);
+      selection.addRange(range);
+    }
```

Comment:
This change is so that instead of everything on the webpage getting selected on ctrl+A, only the command does?
1. Not sure when/why but it isn't working each time.
1. I feel this is out of scope. Let's focus on copying.

Reply:
+1